data/human/ships.txt

```diff
@@ -1117,7 +1117,7 @@ ship "Container Transport"
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
 	description "The Container Transport is a newer, more popular configuration of the Bulk Freighter that replaces the outboard loose cargo holds with intermodal containers for more efficient turnaround in larger, better-equipped ports."
-	description "	As with its geared cousin, however, it remains a slow and difficult to maneuver ship, especially when loaded down with cargo. Some freight caravans use ordinary freighters instead of bulk freighters just to avoid the hassle of constantly having to wait for the Bulk Freighter to properly align itself for hyperspace travel."
+	description "	As with its geared cousin, however, it remains a slow and difficult to maneuver ship, especially when loaded down with cargo. Some freight caravans use ordinary freighters instead of container transports just to avoid the hassle of constantly having to wait for the Container Transport to properly align itself for hyperspace travel."
```

Comment:
It seems odd to me that the first use of "container transport" in this sentence wouldn't be capitalized, but the second would. It's also capitalized in the first line of the description. We should probably capitalize it in both locations. The same change should be made to the Bulk Freighter.

Reply:
> It seems odd to me that the first use of "container transport" in this sentence wouldn't be capitalized, but the second would. It's also capitalized in the first line of the description. We should probably capitalize it in both locations. The same change should be made to the Bulk Freighter.

Makes sense to me